build-system/compile/bundles.config.extensions.json

```diff
@@ -199,6 +199,12 @@
   },
   {"name": "amp-facebook-like", "version": "0.1", "latestVersion": "0.1"},
   {"name": "amp-facebook-page", "version": "0.1", "latestVersion": "0.1"},
+  {
+    "name": "amp-facebook-page",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {"hasCss": true}
+  },
```

Comment:
You may need to pull from `main`, and run this command since the reformatting change has been merged:

```console
amp prettify --fix --files=build-system/compile/bundles.config.extensions.json
```